services/api-server/src/simcore_service_api_server/services/rabbitmq.py

```diff
@@ -14,6 +21,10 @@ def setup_rabbitmq(app: FastAPI) -> None:
     app.state.rabbitmq_client = None
     app.state.log_distributor = None
 
+    instrumentator: Instrumentator | None = None
+    if app.state.settings.API_SERVER_PROMETHEUS_INSTRUMENTATION_ENABLED:
```

Comment:
if disabled, I see that `app.state.health_checker`  is never set which will eventually raise an exception in the health check entrypoint since `get_health_checker` raises which is probably converted to 5XX

Reply:
Very good point. I have factored out the prometheus intrumentation from the health check. That means the health checker should always be there, so `get_health_checker` should not raise an AssertionError.